HALO2.md

```diff
@@ -0,0 +1,23 @@
+### Halo2 configuration
+
+We currently support Halo2 circuits in an experimental stage. To use Halo2, you need to set the `kind` to `halo2` in the `mopro-config.toml` file.
+
+```toml
+[circuit]
+kind = "halo2" # Options: circom, halo2
```

Comment:
Could we rename this option to `adapter`? https://zkmopro.org/docs/intro

Reply:
Agree. Thank you for pointing out.